sdk/python/kfp/dsl/_metadata.py

```diff
@@ -16,6 +16,7 @@
 from abc import ABCMeta, abstractmethod
 from .types import BaseType, _check_valid_type_dict, _instance_to_dict
 
+
```

Comment:
Nit. It's better to do style changes and refactorings in separate PRs so that the main PR diff is smaller.
One way is to create separate commits, tag them with branches and make PRs:
```
master
|
V
Fixed the style -> fix-style branch -> Fix style PR
|
V
Added feature -> add-feature branch -> Added feature PR
```

GitHub allows seeing the diffs of individual commits.

Reply:
sure.